test/extended/builds/build_pruning.go

```diff
@@ -149,4 +152,34 @@ var _ = g.Describe("[builds][pruning] prune builds based on settings in the buil
 
 	})
 
+	g.It("buildconfigs should have a default history limit set when created via the group api", func() {
```

Comment:
we run these tests nightly and check the results, i can live w/ it for now at least.